src/poliastro/plotting/core.py

```diff
@@ -145,14 +147,14 @@ def show(self):
 class OrbitPlotter3D(_PlotlyOrbitPlotter):
     """OrbitPlotter3D class."""
 
-    def __init__(self, figure=None, dark=False, *, num_points=150, plane=None):
-        super().__init__(figure, num_points=num_points, plane=plane)
+    def __init__(self, figure=None, dark=False, *, num_points=150, plane=None, unit=u.km):
+        super().__init__(figure, num_points=num_points, plane=plane, unit=unit)
         self._layout = Layout(
             autosize=True,
             scene=dict(
-                xaxis=dict(title="x (km)"),
-                yaxis=dict(title="y (km)"),
-                zaxis=dict(title="z (km)"),
+                xaxis=dict(title="x f"({self._unit})""),
```

Comment:
This is a syntax error, it should be:

```suggestion
                xaxis=dict(title=f"x ({self._unit})"),
```

and so forth